index.html

```diff
@@ -12,20 +12,27 @@
 
   <header>
     <nav>
-      <ul class="nav-bar">
+      <ul class="nav-bar desktop-nav-items">
         <li class="logo">Logo</li>
```

Comment:
- [ ] Please, change the logo name to your proper name, I think it should be fine to let that logo if maybe that one was an image or a text logo, but that name logo does not mean anything there, Please change it.